systems/registry/network/github.com/ukama/ukama/systems/registry/network/pb/gen/network.pb.go

```diff
@@ -0,0 +1,1535 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
this file should be removed because  generated for the wrong location path.

Reply:
removed